docs/content/architecture/pgadmin4.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "pgAdmin 4"
```

Comment:
🤔 I can't tell if the upstream project is called "pgAdmin" or "pgAdmin 4"... The website and docs seem to use both all over. Do we want this URL to be `.../pgadmin4` long-term, or should we hedge and have it be `.../pgadmin`?

Either way, I'm fine with the title being pgAdmin 4.

https://www.pgadmin.org/faq/
> **What is pgAdmin 4?**
> pgAdmin is …
>
> **What browsers does pgAdmin support?**
> The following browsers are supported with pgAdmin…

Reply:
If `pgadmin` works, I would prefer that, but I don't have any strong feelings either way.